docs/source/onedal/build_app/build-application.rst

```diff
@@ -17,21 +17,81 @@
 Build applications with oneDAL
 ==============================
 
-See Get Started Guides for `Windows\*
-<https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-daal-for-windows/top.html>`_
-and `Linux\* <https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-daal-for-linux/top.html>`_
-for instruction on how to build applications for C++.
 This section contains instructions for building applications with |short_name| for SYCL\*.
 
-- :ref:`app_on_win`
 - :ref:`app_on_lin`
+- :ref:`app_on_win`
+
+.. _app_on_lin:
+
+Applications on Linux* OS
+-------------------------
+
+#. Install |short_name|.
+
+#. Set environment variables by calling ``<install dir>/setvars.sh``.
+
+#. Build your application with clang++:
+
+   - Add ``fsycl`` option to the command:
```

Comment:
@makart19 @amgrigoriev @vlad-nazarov @KulikovNikita  - what would  be correct instructions?

I think we have to replace clang++ with dpcpp call without fsycl. Also don;t sure that "ONEAPI_DAAL_USE_MKL_GPU_GEMM" is required

Reply:
@makart19 @amgrigoriev @vlad-nazarov @KulikovNikita can anyone comment on this?

Reply:
@makart19 @amgrigoriev @vlad-nazarov @KulikovNikita can anyone comment on this?

Reply:
I think it is better to replace clang++ with dpcpp and remove -fsycl flag. Our examples we build in this way.

Reply:
I didn't find any place in our code where "ONEAPI_DAAL_USE_MKL_GPU_GEMM" export would be used.

Reply:
Let me check the relevance of the entire instruction within a few days. We have added the libonedal_dpc library, which is not listed here

Reply:
@vlad-nazarov  please let me know when you check the instructions

Reply:
@outoftardis, updated the Linux example. Please, check. Maybe should we do the same for windows? I think windows instruction not actual too...

Reply:
@vlad-nazarov @makart19 @KulikovNikita @amgrigoriev @napetrov who could tell whether any changes are needed for Windows instructions?